airbyte-integrations/connectors/source-sendgrid/source_sendgrid/schemas/email_activity.json

```
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "from_email": {
+      "type": "string"
+    },
+    "msg_id": {
+      "type": "string"
+    },
+    "subject": {
+      "type": "string"
+    },
+    "to_email": {
+      "type": "string"
+    },
+    "status": {
+      "type": "string"
+    },
+    "opens_count": {
+      "type": "integer"
+    },
+    "clicks_count": {
+      "type": "integer"
+    },
+    "last_event_time": {
+      "type": "string"
```

```suggestion
      "type": "string",
      "format": "date-time"
```
This will allow the downstream destinations to consider this field as a date and use proper timestamp column types. According to Sendgrid's API reference, this field is in iso 8601 format so no additional transformation from your side should be required.